src/sentry/static/sentry/app/views/issueList/overview.tsx

```diff
@@ -201,45 +248,42 @@ const IssueListOverview = createReactClass({
     // cause weird side effects (e.g. ProcessingIssueList mounting and making
     // a request, but immediately unmounting when fetching saved searches)
     resetSavedSearches();
-  },
+  }
 
-  // Memoize projects fetched as selections are made
-  // This data is fed into the action toolbar for release data.
-  projectCache: {},
+  private _poller: any;
+  private _lastRequest: any;
+  private _streamManager = new StreamManager(GroupStore);
 
-  getQuery() {
+  getQuery = (): string => {
```

Comment:
We usually use class properties with arrow functions only if it's the handler so that the scope is right. In cases like this the `getQuery() {` is preferred.